src/main/kotlin/us/ihmc/build/IHMCBuildPlugin.kt

```diff
@@ -122,5 +125,7 @@ class IHMCBuildPlugin : Plugin<Project>
       // IHMCBuildTools.defineExtraSourceSetCompositeTask("publishExtraSourceSets", arrayListOf("publish"), project)
 
       IHMCBuildTools.defineDynamicCompositeTask(project)
+
+      ciPlugin.apply(project);
```

Comment:
This is done because now, ihmc-ci is not a stand alone plugin. Rather, apart of ihmc-build plugin. I did this in order to have the least amount of code changes.